Sources/TuistKit/GraphMappers/TuistWorkspaceIdentifierMapper.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import TuistCore
+import TuistSupport
+
+final class TuistWorkspaceIdentifierMapper: WorkspaceMapping {
```

Comment:
I took the opportunity to move this here as it nicely fits into this concept :)

Reply:
That's super nice! I'd just add a little bit of documentation as it's not really obvious from first looking at it what htis does 🙂

Reply:
Good idea, updated the doc comments for this mapper